services-directions-models/src/test/resources/route_options_v5.json

```diff
@@ -7,6 +7,7 @@
   "language": "ru",
   "radiuses": ";unlimited;5.1",
   "bearings": "0,90;90,0;",
+  "layers": "-42;;0",
```

Comment:
Could you also add this to the other file? Refs https://github.com/mapbox/mapbox-java/issues/1286.

Reply:
done